java-checks/src/test/files/checks/SillyEquals.java

```diff
@@ -0,0 +1,70 @@
+public class MyClass {
+
+  public interface Interface {
+  }
+
+  Object object;
+  Object[] objects;
+  Integer integer;
+  Integer[] integers;
```

Comment:
please name more differently the array and the instance. This naming make the test hard to read.